subsystems/drivebase/SwerveDrive.java

```diff
@@ -39,78 +43,103 @@ public abstract class SwerveDrive extends DriveBase {
+        kModuleConfigBackRight.setDriveCurrentLimit(config.kMaxDriveCurrent);
+        kModuleConfigBackRight.setSteerCurrentLimit(config.kMaxSteerCurrent);
+        kModuleConfigBackRight.setSteerEncoderType(config.kBackRightModuleEncoderType);
 
         ShuffleboardTab tab = Shuffleboard.getTab("Drivetrain");
         mModules = new SwerveModule[4];
 
-        // front left
-        mModules[0] = Mk4iSwerveModuleHelper.createNeo(
-                tab.getLayout("Front Left Module", BuiltInLayouts.kList).withSize(2, 4)
-                        .withPosition(0, 0),
-                kModuleConfig, config.kSwerveModuleGearRatio, config.kFrontLeftModuleDriveMotor,
-                config.kFrontLeftModuleSteerMotor, config.kFrontLeftModuleSteerEncoder,
-                config.kFrontLeftModuleSteerOffset);
+       // front left
+       
+       mModules[0] = Mk4iSwerveModuleHelper.createNeo(
+        tab.getLayout("Front Left Module", BuiltInLayouts.kList).withSize(2, 4)
+                .withPosition(0, 0),
+        kModuleConfigFrontLeft, config.kSwerveModuleGearRatio, config.kFrontLeftModuleDriveMotor,
+        config.kFrontLeftModuleSteerMotor, config.kFrontLeftModuleSteerEncoder,
+        config.kFrontLeftModuleSteerOffset);
 
         // front right
         mModules[1] = Mk4iSwerveModuleHelper.createNeo(
                 tab.getLayout("Front Right Module", BuiltInLayouts.kList).withSize(2, 4)
                         .withPosition(2, 0),
-                kModuleConfig, config.kSwerveModuleGearRatio, config.kFrontRightModuleDriveMotor,
+                kModuleConfigFrontRight, config.kSwerveModuleGearRatio, config.kFrontRightModuleDriveMotor,
                 config.kFrontRightModuleSteerMotor, config.kFrontRightModuleSteerEncoder,
                 config.kFrontRightModuleSteerOffset);
 
         // back left
         mModules[2] = Mk4iSwerveModuleHelper.createNeo(
                 tab.getLayout("Back Left Module", BuiltInLayouts.kList).withSize(2, 4)
                         .withPosition(4, 0),
-                kModuleConfig, config.kSwerveModuleGearRatio, config.kBackLeftModuleDriveMotor,
+                kModuleConfigBackLeft, config.kSwerveModuleGearRatio, config.kBackLeftModuleDriveMotor,
                 config.kBackLeftModuleSteerMotor, config.kBackLeftModuleSteerEncoder,
                 config.kBackLeftModuleSteerOffset);
 
         // back right
         mModules[3] = Mk4iSwerveModuleHelper.createNeo(
                 tab.getLayout("Back Right Module", BuiltInLayouts.kList).withSize(2, 4)
-                        .withPosition(6, 0),
-                kModuleConfig, config.kSwerveModuleGearRatio, config.kBackRightModuleDriveMotor,
-                config.kBackRightModuleSteerMotor, config.kBackRightModuleSteerEncoder,
-                config.kBackRightModuleSteerOffset);
+                    .withPosition(6, 0),
+            kModuleConfigBackRight, config.kSwerveModuleGearRatio, config.kBackRightModuleDriveMotor,
```

Comment:
I'm confused why these changes are part of this PR. Aren't they already in the 2024 branch?